beeai/README.md

```diff
@@ -6,13 +6,13 @@ Every agent can run individually or pick up tasks from a Redis queue.
 ## Architecture
 
 Three agents process tasks through Redis queues:
-- **Triage Agent**: Analyzes JIRA issues and determines resolution path
-- **Rebase Agent**: Updates packages to newer upstream versions
-- **Backport Agent**: Applies specific fixes/patches to packages
+- **Triage Agent**: Analyzes JIRA issues and determines resolution path. It can ask for clarification, create tasks for other agents or may take no action if not needed.
+- **Rebase Agent**: Updates packages to newer upstream versions. It looks for upstream references that are linked, attached and present in the description or comments in the issue.
+- **Backport Agent**: Applies specific fixes/patches to packages. It looks for patches that are linked, attached and present in the description or comments in the issue.
```

Comment:
I think it would be beneficial to include what steps do the agents take, like explain what information it uses for triaging and how it tries to find for fixes and the whole logic behind it; that's something that we are getting asked often when meeting with maintainers. 

Same would apply to rebase and backport, e.g. in backport agent it would be good to describe the methodology how it tries to apply the new patch.

Reply:
agreed! literally got asked about this yesterday, so now we can easily forward anyone to the readme